registry/txt.go

```diff
@@ -331,32 +347,41 @@ func extractRecordType(name string) (baseName, recordType string) {
 	return name, ""
 }
 
-// dropAffix strips TXT record to find an endpoint name it manages
-// It takes into consideration a fact that it could contain record type
-// So it gets stripped first
-func (pr affixNameMapper) dropAffix(name string) string {
+// dropAffixExtractType strips TXT record to find an endpoint name it manages
+// it also returns the record type
+func (pr affixNameMapper) dropAffixExtractType(name string) (baseName, recordType string) {
```

Comment:
There is no use of named return values, so we can also just do:
`(baseName, recordType string)` -> `(string, string)`

If there is a blank return then we would return `"",""` in the current version

Reply:
The problem with `(string, string)` is it makes it difficult to know what the strings contain. The names convey the semantics.

Reply:
This could be also a comment instead of named return values, which actually define variables in scope of the func.

Reply:
Named return values work better with IDEs than comments do.